zerver/models.py

```diff
@@ -1606,6 +1606,25 @@ class UserBaseSettings(models.Model):
     send_private_typing_notifications = models.BooleanField(default=True)
     send_read_receipts = models.BooleanField(default=True)
 
+    # Who in the organization has access to users' actual email
+    # addresses.  Controls whether the UserProfile.email field is the
+    # same as UserProfile.delivery_email, or is instead garbage.
```

Comment:
Let's replace "is instead garbage." with "is instead a fake generated value encoding the user ID and realm hostname."

Maybe also saying "the deprecated UserProfile.email field" would be good.

Reply:
Are we okay to mention "depreceated" just here in the comment and not in API docs.